src/main/java/xyz/groundx/caver_ext_kas/kas/kip37/KIP37.java

```diff
@@ -0,0 +1,3053 @@
+     * @param id The token id to mint additionally.
+     * @param amount The token amount to mint additionally.
+     * @return Kip37TransactionStatusResponse
+     * @throws ApiException
+     */
+    public Kip37TransactionStatusResponse mint(String addressOrAlias, String to, BigInteger id, BigInteger amount) throws ApiException {
+        return mint(addressOrAlias, to, id, amount, null);
+    }
+
+    /**
+     * Mint token for a given KIP-37 contract. Minting is possible after having created a token with {@link KIP37#create(String, BigInteger, BigInteger, String, String)}<br>
+     * It sets a sender to contract deployer.<br>
+     * POST /v1/contract/{contract-address-or-alias}/token/mint
+     *
+     * <pre>{@code
+     * String contractAddress = "0x{contractAddress}";
+     * String to = "0x{toAddress}";
+     * String tokenId = "0x01";
+     * String amount = "0x01";
+     *
+     * Kip37TransactionStatusResponse response = caver.kas.kip37.mint(contractAddress, to, tokenId, amount);
+     * }</pre>
+     *
+     * @param addressOrAlias The contract address(in hex, with the 0x prefix) or alias.
+     * @param to The account address of the token owner.
+     * @param id The token id to mint additionally.(in hex, with the 0x prefix)
+     * @param amount The token amount to mint additionally.(in hex, with the 0x prefix)
+     * @return Kip37TransactionStatusResponse
+     * @throws ApiException
+     */
+    public Kip37TransactionStatusResponse mint(String addressOrAlias, String to, String id, String amount) throws ApiException {
+        return mint(addressOrAlias, to, id, amount, null);
+    }
+
+    /**
+     * Mint token for a given KIP-37 contract. Minting is possible after having created a token with {@link KIP37#create(String, BigInteger, BigInteger, String, String)}<br>
+     * POST /v1/contract/{contract-address-or-alias}/token/mint
+     *
+     * <pre>{@code
+     * String contractAddress = "0x{contractAddress}";
+     * String to = "0x{toAddress}";
+     * String sender = "0x{senderAddress}";
+     * BigInteger tokenId = BigInteger.valueOf(1);
+     * BigInteger amount = BigInteger.valueOf(1);
+     *
+     * Kip37TransactionStatusResponse response = caver.kas.kip37.mint(contractAddress, to, tokenId, amount, sender);
+     * }</pre>
+     *
+     * @param addressOrAlias The contract address(in hex, with the 0x prefix) or alias.
+     * @param to The account address of the token owner.
+     * @param id The token id to mint additionally.
+     * @param amount The token amount to mint additionally.
+     * @param sender The account address to mint token additionally.
+     * @return Kip37TransactionStatusResponse
+     * @throws ApiException
+     */
+    public Kip37TransactionStatusResponse mint(String addressOrAlias, String to, BigInteger id, BigInteger amount, String sender) throws ApiException {
+        return mint(addressOrAlias, to, new BigInteger[]{id}, new BigInteger[]{amount}, sender);
```

Comment:
```suggestion
    public Kip37TransactionStatusResponse mint(String addressOrAlias, String to, BigInteger id, BigInteger amount, String minter) throws ApiException {
        return mint(addressOrAlias, to, new BigInteger[]{id}, new BigInteger[]{amount}, minter);
```
Can we use `minter` and `pauser` instead of `from` or `sender`?

Reply:
I modified it. Would you check it?